Pyrado/pyrado/algorithms/step_based/sac.py

```diff
@@ -161,6 +162,7 @@ def __init__(
             max_grad_norm=max_grad_norm,
             num_workers=num_workers,
             logger=logger,
+            use_trained_policy_for_refill=use_trained_policy_for_refill,
```

Comment:
Please do the same for DQL

Reply:
Done in fdbb47d708b088e103a979cf831a2a711c0fc77e